cx_Freeze/freezer.py

```diff
@@ -331,38 +367,11 @@ def _GetDependentFiles(self, path):
                         dependentFile = dependentFile[:pos].strip()
                     if dependentFile:
                         dependentFiles.append(dependentFile)
-                if sys.platform == "darwin":
-                    # Make library paths absolute. This is needed to use
-                    # cx_Freeze on OSX in e.g. a conda-based distribution.
-                    # Note that with @rpath we just assume Python's lib dir,
-                    # which should work in most cases.
-                    dependentFiles = [p.replace('@loader_path', dirname)
-                                      for p in dependentFiles]
-                    dependentFiles = [p.replace('@rpath', sys.prefix + '/lib')
-                                      for p in dependentFiles]
-            if sys.platform == "darwin":
-                dependentFiles = [self._CheckDependentFile(f, dirname)
-                    for f in dependentFiles if self._ShouldCopyFile(f)]
-            else:
-                dependentFiles = [os.path.normcase(f)
-                    for f in dependentFiles if self._ShouldCopyFile(f)]
+
+            dependentFiles = [f for f in dependentFiles if self._ShouldCopyFile(f)]
```

Comment:
Here you're partially breaking my PR #640 - os.path.normcase(f) - in windows this is essential.